iFixit/react-commerce
# Add product page cross sell section

closes #679 

This PR adds the cross sell section. To be able to open the PR I also had to update Jest tests to take environment variables into account, that's why you see changes related to env variables and jest files.

## QA

1. Open Vercel preview
2. Visit a product with cross sell items (e.g. https://react-commerce-lesb44g2v-ifixit.vercel.app/Products/apple-watch-38-mm-series-2-replacement-battery)
3. Verify that you can select and deselect items
4. Verify that you can add to cart
